Named_Boxarts/Driver 2 - Back on the Streets (Spain) (Disc 2) (Rev 1).png

```diff
@@ -0,0 +1 @@
+Driver 2 - Back on the Streets (Europe).png
```

Comment:
There can be cases where the Spain boxart is different than the Europe one since it's a different language. While I'm okay with this change, I'd like other's opinions first. Thanks!

Reply:
I'm fine too, I tent to use the European box when an specific contry release is har to find or the one that exists is in a pretty bad quality form, but if it exists I think is better to provide it rather than tide them to a generic, often UK only one.